docs/data/date-pickers/date-picker/examplesConfig.ts

```diff
@@ -195,6 +203,44 @@ export const datePickerExamples: PickersSubcomponentType = {
     },
     slots: ['root', 'today'],
   },
+  PickersMonth: {
+    examples: {
+      customTheme: {
+        type: 'success',
+        componentProps: { views: ['month'] },
+      },
+      sxProp: {
+        type: 'warning',
+        parentSlot: 'desktopPaper',
+        comments:
+          'Because of the structure of the DesktopDatePicker, the `sx` prop needs to be applied to the `desktopPaper` slot',
+        componentProps: { views: ['month'] },
+      },
+
+      styledComponents: {
+        type: 'warning',
+        comments:
+          'You will need to specify the `disablePortal` prop for the popper in order to be able to use `DesktopDatePicker` as a styled component ',
+        componentProps: { views: ['month'] },
+      },
+    },
+    slots: ['root', 'monthButton'],
+  },
+  InputBase: {
```

Comment:
Is there a particular reason why you have chosen this component? 🤔 
It is a low-level `@mui/material` primitive and styling it would impact every input in the application. 🙈 
I'd probably avoid the conflict of interest for this issue and skip introducing it without more though of how to best propose it.
Maybe we even should have the `TextField` styled so that we could override it via the `styleOverrides`... 🤷